src/dice/mempool.c

```diff
@@ -94,6 +103,9 @@ mempool_alloc(size_t n)
     size_t size     = n + sizeof(entry_t);
     unsigned bucket = _bucketize(size);
     size            = _sizes[bucket];
+    #ifdef ALLOC_STRICT_ALIGN8
```

Comment:
You can always use the align_up macro and redefine it to by identity when the alignment is not set.